README.md

```diff
@@ -32,29 +69,18 @@ $keypair = Keypair::fromStrings(
 );
 
 // Replace this with what you received from the app
-$apiKey = 'foobarbaz';
+$apiKey = 'your-api-key';
 
-$debugMode = true;
-$environment = new Sandbox($debugmode);
+$debugMode = ture;
```

Comment:
This should be true, if running from the commandline, it will output debug information from request/response